src/Sarif/Autogenerated/AnnotatedCodeLocation.cs

```diff
@@ -84,12 +84,18 @@ public SarifNodeKind SarifNodeKind
         public string Message { get; set; }
 
         /// <summary>
-        /// Describes the location.
+        /// Categories the location.
         /// </summary>
         [DataMember(Name = "kind", IsRequired = false, EmitDefaultValue = false)]
         public AnnotatedCodeLocationKind Kind { get; set; }
 
         /// <summary>
+        /// Classifies state transitions in code locations relevant to a taint analysis.
+        /// </summary>
+        [DataMember(Name = "taint", IsRequired = false, EmitDefaultValue = false)]
+        public AnnotatedCodeLocationTaint Taint { get; set; }
```

Comment:
Yes, I remember our discussion about this: that a taint could occur at an `annotatedCodeLocation` with any `kind` (although it would most likely be a `call`), and therefore that we should represent tainting independently from `kind`.

But I think the SARIF property should be called `taintKind`, and it should be represented in the SDK by an enumerated type `TaintKind`.